vulnerabilities/api.py

```diff
@@ -254,6 +265,17 @@ def bulk_search(self, request):
 
         return Response(response)
 
+    @action(detail=False, methods=["get"])
+    def all(self, request):
+        vulnerable_packages = Package.objects.filter(
+            packagerelatedvulnerability__fix=False
+        ).distinct()
```

Comment:
Can you confirm this QS triggers only 1 DB query? It should be asserted in your unit test as this is critical for the performances of those type of specialized actions.

Reply:
Asserted the number of queries, only one query is used for getting all vulnerable packages